Shared/Cli/OperationArguments.cs

```diff
@@ -726,32 +743,46 @@ internal virtual void CreateTargetUri()
 
             queryUrl = buffer.ToString();
 
+            // If the actual-url override has been set, honor it.
+            if (!string.IsNullOrEmpty(_urlOverride))
+            {
+                if (!Uri.TryCreate(_urlOverride, UriKind.Absolute, out Uri uri))
+                {
+                    Trace.WriteLine($"failed to parse \"{_urlOverride}\", unable to set URL override.");
+                }
+
+                actualUrl = uri.ToString();
```

Comment:
This requires an `else` clause, otherwise the `.TryCreate` could fail and the `.ToString()` would still happen - that's bad.